interface/wx/aui/floatpane.h

```diff
@@ -0,0 +1,33 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/aui/floatpane.h
+// Purpose:     wxaui: wx advanced user interface - docking window manager
+// Author:      Benjamin I. Williams
+// Modified by:
```

Comment:
Minor, but we shouldn't/don't use this line in new files any more, it doesn't make much sense when `git blame` is always available.